### PR TITLE
DOCS-7334 small text update

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-7334 small text update

(Small update since it's really API Manager that does the comparison, and we don't want to confuse the reader with that detail.)

### CODE REVIEW

**Review by smulesoft — APPROVED** [review]
de-niro-travis-clap

File: modules/ROOT/pages/to-request-access.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ The OAuth 2 grant type:
 
 == API Groups
 
-When you request access to an API group, Exchange automatically compares all available contracts and applies the least restrictive contract. The application uses that contract to access the API group instance.
+When you request access to an API group, the least restrictive of the available contracts is applied. The application uses that contract to access the API group instance.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-request-access.adoc]: A similar suggestion as before to simplify the wording: "...the least restrictive available contract is applied."

**Review by dhirsch1 — APPROVED** [review]
One comment

File: modules/ROOT/pages/to-request-access.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-request-access.adoc]: Agreed and fixed!
